modules/cnv-creating-bridge-nad-cli.adoc

```diff
@@ -66,19 +66,6 @@ spec:
 ----
 <1> You must substitute the actual `name` value from the
 NetworkAttachmentDefinition.
-+
```

Comment:
Can we add a note explaining that the NetworkAttachmentDefinition is available only to VMs within the same Namespace?